src/components/similar_movies.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getFullMovieDetails, getMovieSuggestions } from 'actions';
```

Comment:
I think that these actions are redundant, they are never used within the component.